<launch>
        <node pkg="tf"
	      type="static_transform_publisher" 
	      name="localizer" 
	      args="0 0 0 0 0 0 map odom 100" 
	/>
  <param name="/use_sim_time" value="true"/>

  <!-- Launch stage world -->
  <include file="$(find stage_worlds)/launch/kinect_world.launch" />
	
  <!-- Arbiter -->
  <node name="arbiter" pkg="stage_behavior" type="arbiter">
    <param name="inputs" value="3" />
    <param name="publish_rate" value="10" />
  </node>

  <!-- Escape behavior -->
  <node name="escape" pkg="stage_behavior" type="escape_behavior">
    <remap from="scan" to="base_scan"/>
    <remap from="cmd_vel" to="cmd_vel1"/>
    <param name="publish_rate" value="20"/>
  	<param name="backoff_duration" value="1.0" />
  	<param name="backoff_velocity" value="0.5" />
  	<param name="turn_duration" value="1.0" />
  	<param name="turn_velocity" value="1.5" />
    <param name="bump_distance" value="1.0" />
    <param name="robot_size" value="0.33" />
    <param name="obstacle_detected" value="false" />
  </node>
 <!-- Cruise behavior -->
  <node name="cruise" pkg="stage_behavior" type="cruise_behavior">
    <remap from="cmd_vel" to="cmd_vel3"/>
    <param name="publish_rate" value="10"/>
    <param name="cruise_velocity" value="0.5"/>
  </node>
  <!-- Other behavior -->

  <node name="seek" pkg="stage_behavior" type="seek_behaviour" output="screen"
		launch-prefix="xterm -e">
    <remap from="cmd_vel" to="cmd_vel0"/>
    <param name="publish_rate" value="10"/>
  </node> 

 
</launch>

